doc/dependencies.rst

```diff
@@ -0,0 +1,27 @@
+.. _dependencies:
+
+:octicon:`package` Dependencies
+===============================
+
+Run Time Dependencies
+---------------------
+
+================================= ================================= ================
+Dependency                        Purpose                           License
+================================= ================================= ================
+`Python 3`_                       Python version 3.6.1 and above    PSF
```

Comment:
Is it really Python 3.6?
The Github workflows test against 3.9 and later....

Reply:
Could we maybe think about generating the list of dependencies?
I think manual maintenance of such lists is quite tedious and error-prone. 
For the java projects project-keeper does that already.

Reply:
This was just converted from `md` to `rst`. Yes, we should include it in the toolbox. In fact, some basic work has already been done: https://github.com/exasol/python-toolbox/pull/76/files.